pkgs/top-level/python-packages.nix

```diff
@@ -13923,7 +13924,9 @@ self: super: with self; {
 
   torchvision = callPackage ../development/python-modules/torchvision { };
 
-  torchvision-bin = callPackage ../development/python-modules/torchvision/bin.nix { };
+  torchvision-bin = callPackage ../development/python-modules/torchvision/bin.nix {
+    cudaPackages = pkgs.cudaPackages_12;
```

Comment:
Does torchvision-bin not work with `pkgs.cudaPackages`?

Reply:
No, torchvision now needs cudart 12. This is why I have added this line.

Reply:
gotcha... good motivation for us to complete the cudaPackages 11 -> 12 upgrade :P